main.cpp

```diff
@@ -91,6 +92,8 @@ int main(int argc, char** argv)
             untrusted_dir = get_dir(i++);
         else if (a == "--experimental")
             ;
+        else if (a.rfind("-D", 0) == 0)
```

Comment:
Would /D work on MSVC?

Reply:
Yes, cl.exe support both /D and -D. But I would suggest we just stick with -D for the oeedger8r parameters since it works on both.

Reply:
Great. I like -D